doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -984,10 +984,11 @@ Helper tactics
 
 .. tacn:: decide equality
 
-   Solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}`,
-   where :g:`R` is an inductive type such that its constructors do not take
-   proofs or functions as arguments, nor objects in dependent types. It
-   solves goals of the form :g:`{x = y} + {~ x = y}` as well.
+   Solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}` or
+   :g:`forall x y : R, (x = y) \/ (~ x = y)`, where :g:`R` is an inductive type
+   such that its constructors do not take proofs or functions as arguments, nor
+   objects in dependent types. It solves goals of the form
+   :g:`{x = y} + {~ x = y}` or :g:`(x = y) \/ (~ x = y)` as well.
```

Comment:
I've hard wrapped it for you:
```suggestion
   Solves a goal of the form :g:`forall x y : R, {x = y} + {~ x = y}` or
   :g:`forall x y : R, (x = y) \/ (~ x = y)`, where :g:`R` is an
   inductive type such that its constructors do not take proofs
   or functions as arguments, nor objects in dependent types. It
   solves goals of the form
   :g:`{x = y} + {~ x = y}` or :g:`(x = y) \/ (~ x = y)` as well.
```

Reply:
I'm sorry, did I miss the maximum line size somewhere in the contributor documentation?  I've already wrapped it using the emacs macro (M-q).

Reply:
It's not particularly important for the documentation, however it does make reviewing on Github easier. I also wrapped using emacs, but mine seems to have done it tighter (I'm guessing this is 80).

Reply:
OK.